starter-code/src/functions-and-arrays.js

```diff
@@ -1,6 +1,35 @@
 // Find the maximum
+function maxOfTwoNumbers(a,b) {
+  if(a > b){
+    //console.log(a + "is the larger number.");
+    return a;
+  }else if(a < b){
+    //console.log( b + "is the largest number")
+    return b;
+  }else{
+    //console.log("Numbers are equal.")
+  } return a,b; 
```

Comment:
3 things:
- the last return `return a, b` is outside the `else`
- `return a,b` is a interesting statement. It will only return the last expression. The before ones it will be evaluated.
- the logic can be simplified to:
```
function maxOfTwoNumbers (a, b) {
  if (a > b) {
   return a;
  }
  return b;
}
```
Because if the `if` is true, it will return and stop the function. Else it will return the another one.